### PR TITLE
feat: Wallet Responsive Web Develop

### PR DESCRIPTION
**Pull request by juntaepark**

### CODE REVIEW

File: packages/web/src/components/wallet/asset-list-header/AssetListHeader.styles.ts [hunk 1]
```diff
@@ -1,22 +1,60 @@
...
   }
+
   .right-section {
     ${mixins.flexbox("row", "center", "center")};
-    margin-left: auto;
+    gap: 36px;
   }
   .assets-search {
-    margin-left: 36px;
+    // margin-left: 36px;
```
Comment (jinoosss) [inline packages/web/src/components/wallet/asset-list-header/AssetListHeader.styles.ts:58]: Remove unnecessary comments.

File: packages/web/src/components/wallet/asset-list-header/AssetListHeader.styles.ts [hunk 2]
```diff
@@ -1,22 +1,60 @@
...
   h2 {
     ${fonts.h5};
     color: ${({ theme }) => theme.color.text02};
-    margin-right: 36px;
+    ${media.mobile} {
+      ${fonts.h6};
+    }
   }
+
   .right-section {
```
Comment (jinoosss) [inline packages/web/src/components/wallet/asset-list-header/AssetListHeader.styles.ts:53]: Text alignment in Switch components is broken on tablet environments.

**Review by jinoosss — COMMENTED** [review]
I left a few comments.

File: packages/web/src/components/wallet/asset-list-header/AssetListHeader.styles.ts (hunk 1 above)
Comment (juntaepark) [reply packages/web/src/components/wallet/asset-list-header/AssetListHeader.styles.ts:58]: reslove !

File: packages/web/src/components/wallet/asset-list-header/AssetListHeader.styles.ts (hunk 2 above)
Comment (juntaepark) [reply packages/web/src/components/wallet/asset-list-header/AssetListHeader.styles.ts:53]: reslove !

**Review by jinoosss — APPROVED** [review]
Approve !

**Pull request merged**